packages/uswds-core/src/styles/mixins/helpers/checkbox-and-radio-colors.scss

```diff
@@ -66,7 +66,7 @@
         color.adjust(color($input-text-color), $alpha: $input-border-alpha);
 
       @media (forced-colors: active) {
-        outline: 2px solid transparent;
+        border: $border-high-contrast;
```

Comment:
I noticed there was a visual regression here. Would switching back to outline would fix it?

![image](https://user-images.githubusercontent.com/93996430/226699779-d0f6f984-018b-4933-a319-9c7ecc72a9a2.png)
![image](https://user-images.githubusercontent.com/93996430/226700016-f6043b37-71f3-4a9c-8844-d537aa14b995.png)